libraries/Update/examples/OTAWebUpdater/OTAWebUpdater.ino

```diff
@@ -7,11 +7,16 @@
 
 #define SSID_FORMAT "ESP32-%06lX"  // 12 chars total
 //#define PASSWORD "test123456"    // generate if remarked
+const char * authUser = "admin";
+const char * authPass = "admin";
```

Comment:
I'd suggest generating a random password that you print to the console to let the user know what it is over using a hard-coded password.